tests/private-active-active/proxy.tf

```diff
@@ -25,7 +26,14 @@ resource "aws_instance" "proxy" {
     aws_security_group.proxy.id,
   ]
 
-  user_data = data.template_cloudinit_config.config_proxy.rendered
+  user_data = base64encode(
+    templatefile(
+      "${path.module}/templates/squidproxy.sh.tpl",
```

Comment:
[This run](https://github.com/hashicorp/terraform-aws-terraform-enterprise/runs/2850073122?check_suite_focus=true) passed when I switched from cloud-init to a bash script.